crossing-training.lic

```diff
@@ -799,6 +799,8 @@ class CrossingTraining
       stop_play
       UserVars.song = @song_list[UserVars.song] || @song_list.first.first
       return play_song?
+    when 'muster the strength to perform'
+      pause 30
```

Comment:
❓ Do we really want to pause 30? Unlike performance, crossing-training is often playing while trying to train something else.

Reply:
good point